5-AccessControl/1-call-api-roles/AppCreationScripts/sample.json

```diff
@@ -79,11 +71,10 @@
         },
         {
           "key": "Enter the application ID (clientId) of the 'TodoListAPI' application copied from the Azure portal",
-          "value": "service.AppId"
+          "value": "client.AppId"
```

Comment:
Update the keys to match the config file

Reply:
done!